### PR TITLE
Added main.py

### PR DESCRIPTION
**Pull request by romechenko**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    cur.executemany("""INSERT INTO Boat(model,city,highway,passengers,cargo,towing,msrp) VALUES (%(model)s,%(city)s,%(highway)s,%(passengers)s,%(cargo)s,%(towing)s,%(msrp)s)""", pickupdict)
+
+def connect():
+    try:
+        conn = psycopg2.connect(dbname="postgres", user="postgres")
+        cur = conn.cursor()
+        return cur
+    except:
+        try: 
+            conn = psycopg2.connect(dbname="postgres", user="ijohndoe")
```
Comment (jpfischbein) [inline main.py:24]: What about me? jphelps?

File: main.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        print "   ", row[0] 
+
+def main():      
+    try:
+        cur = connect()
+        createPickupsTable(cur)
+        initDB(cur)
+        printPickups(cur)
+    except:
+        print "Fbase"
```
Comment (jpfischbein) [inline main.py:49]: Nice error message. Hopefully we never see it!

**Review by jpfischbein — CHANGES_REQUESTED** [review]
I think my username should be added to connect.

File: main.py (hunk 1 above)
Comment (romechenko) [reply main.py:24]: ah, I gottchu.

File: main.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        cur = conn.cursor()
+        return cur
+    except:
+        try: 
+            conn = psycopg2.connect(dbname="postgres", user="ijohndoe")
+            cur = conn.cursor()
+            return cur
+        except:
+            try:
+                conn = psycopg2.connect(dbname="postgres", user="jphelps")
```
Comment (romechenko) [inline main.py:29]: This is for you @jpfischbein

**Review by jpfischbein — APPROVED** [review]

**Review by romechenko — COMMENTED** [review]
This should address your concerns

**Pull request merged**